app/controllers/books_controller.rb

```diff
@@ -4,10 +4,13 @@ class BooksController < ApplicationController
   before_action :find_book, only: :show
 
   def index
-    @pagy, @books = pagy(Book.all.order(:created_at))
+    @facade = ::Books::IndexFacade.new
+    @pagy, @books = pagy(@facade.books) # переробити
```

Comment:
comments in english

Reply:
move @pagy and @books to facade as methods

Reply:
you'll have `def all_books` private methods (current `books`) and new `def books` already paginated public method